src/nidcpower/system_tests/grpc_server_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "address": "[::1]",
+    "port": 31760,
```

Comment:
FYI, you can configure it to use a dynamic port and read the port number from stdout. That's what InstrumentStudio's GrpcDeviceServerActivationService does.

Reply:
We considered it, but we felt this was safer, particularly with multiple instance of gRPC Device Server being kicked off at the same time.

Reply:
That shouldn't happen. The server specifies a listening port of 0 when calling bind(), which tells the OS to allocate a port from the ephemeral port range. Then the server asks the OS which port it's using and prints that port number to stdout. This should only fail when all of the ports in the ephemeral port range are in use (and there are thousands of these).